app/scripts/views/base.js

```diff
@@ -888,15 +888,13 @@ define(function (require, exports, module) {
       }
 
       if (_.isFunction(handler)) {
-        var args = [].slice.call(arguments, 1);
-
         // If an `arguments` type object was passed in as the first item,
         // then use that as the arguments list. Otherwise, use all arguments.
         if (_.isArguments(args[0])) {
           args = args[0];
         }
 
-        return handler.apply(this, args);
+        return handler.call(this, ...args);
```

Comment:
> Lack of trust in Babel.

Absolutely not, I'm sorry you got that impression.

All I mean is that when there is a choice between letting the JS interpreter do something natively or doing it explicitly in JS code, I prefer the former on the basis that it's probably quicker. It has nothing to do with trust or otherwise in the code that Babel generates.